presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,81 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix disaggregated coordinator resource group aggregation issue related to ``WAITING_FOR_PREREQUISITE`` queries.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory over-counting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in :func:`tdigest_agg`.
+* Add :func:`to_base32` and :func:`from_base32` varbinary functions.
+* Add support for ``TRUNCATE TABLE`` statement.
+* Add support to set timeout on the query analyzer runtime to prevent long running query analysis like complex regular expressions. The timeout duration can be set by the configuration property ``planner.query-analyzer-timeout`` or session property ``query_analyzer_timeout``.
+* Add :func:`trim_array` function.
+* Add a new optimization for showing results for (interactive) distinct limit N as they become available with no buffering.  This can be enabled by setting the session parameter: ``quick_distinct_limit_enabled`` to true.
+* Remove state (PageBuilder) in ``ZIP_WITH`` as it was getting clobbered when used in ``REDUCE_AGG`` context.
+* Change protoc and grpc version in presto-grpc-api module to adapt to linux aarch64.
+* Upgrade zstandard compression to version 1.5.2.2. This can improve about 2% overall CPU with data compressed.
+* Add support for local round robin shuffle to reduce the partial distinct limit output size.
+* Add a Presto connector for Clickhouse with support of username/password authentication
```

Comment:
This should go inside ClickHouse connector section.
https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections

Reply:
done